verify-service/src/main/java/io/mosip/verifyservice/controller/VpRequestController.java

```diff
@@ -0,0 +1,42 @@
+package io.mosip.verifyservice.controller;
+
+import io.mosip.verifycore.dto.authorizationRequest.AuthorizationRequestCreateDto;
+import io.mosip.verifycore.dto.authorizationRequest.AuthorizationRequestCreateResponseDto;
+import io.mosip.verifycore.dto.authorizationRequest.StatusResponseDto;
+import io.mosip.verifycore.enums.SubmissionState;
+import io.mosip.verifycore.spi.VerifiablePresentationRequestService;
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RequestMapping("/vp-request")
+@RestController
+@CrossOrigin(origins = "*")
+public class VpRequestController {
+
+    @Autowired
+    VerifiablePresentationRequestService verifiablePresentationRequestService;
+
+    @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<AuthorizationRequestCreateResponseDto> createVpRequest(@Valid @RequestBody AuthorizationRequestCreateDto vpRequestCreate) {
+        if (vpRequestCreate.getPresentationDefinition() == null) {
```

Comment:
Yes, like mentioned in above comments we will add necessary logic to take either 

presentationDefinitionId or presentationDefinition based on which is present.

What to do in case both are present should presentationDefinitionId take precedence ?